### PR TITLE
Ekman layer test

### PR DESCRIPTION
**Pull request by Zhengyu-Huang**
Ekman layer tests

- [x] test flux boundary condition
- [x] test advection operator 

@yairchn @charleskawczynski @akshaysridhar @haakon-e       feel free to modify and push

### CODE REVIEW

**Comment by jakebolewski** [comment]
@Zhengyu-Huang can you add this as an additional test to run on the cluster?

https://github.com/CliMA/ClimateMachineCore.jl/blob/main/.buildkite/pipeline.yml#L110

File: examples/column/ekman.jl [hunk 1]
```diff
@@ -76,7 +76,7 @@ function adiabatic_temperature_profile(z; T_surf = 300.0, T_min_ref = 230.0)
     ρθ = ρ * θ
 
     u = FT(1.0)
-    v = FT(1.0)
+    v = FT(0.0)
```
Comment (yairchn) [inline examples/column/ekman.jl:79]: I would set these to 
u = FT(ug)
v = FT(vg)

**Review by yairchn — APPROVED** [review]
see minor comments below

File: examples/column/ekman.jl [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+    u_1 = parent(u)[1]
+    v_1 = parent(v)[1]
+    u_wind = sqrt(u_1^2 + v_1^2)
+
+    # u-momentum
+    bcs_bottom = Operators.SetValue(Cd * u_wind * u_1)  # Eq. 4.16
+    bcs_top = Operators.SetValue(FT(ug))  # Eq. 4.18
+    gradc2f = Operators.GradientC2F(top = bcs_top)
+    gradf2c = Operators.GradientF2C(bottom = bcs_bottom)
+    @. du = gradf2c(ν * gradc2f(u)) + f*(v - vg)  # Eq. 4.8
```
Comment (jakebolewski) [inline examples/column/ekman.jl:125]: if you rebase this PR on main, you should be able to use `Operators.Advection()` now

Comment (Zhengyu-Huang) [reply examples/column/ekman.jl]: I added the Advection operator part
